docs/observability/how_to_guides/tracing/mask_inputs_outputs.mdx

```diff
@@ -292,3 +292,317 @@ async def async_function(key: str) -> int:
 ```
 
 These function-level processors take precedence over client-level processors (`hide_inputs` and `hide_outputs`) when both are defined.
+
+## Quick starts
+
+You can combine LangSmith's rule-based masking with various anonymizers to scrub sensitive information from inputs and outputs. In this how-to-guide, we'll cover working with Microsoft Presidio, AWS Comprehend, and regex.
+
+### Microsoft Presidio
+
+Microsoft Presidio is a data protection and de-identification SDK. The implementation below uses Presidio to anonymize inputs and outputs before they are sent to LangSmith. For up to date information, please refer to Presidio's [official documentation](https://microsoft.github.io/presidio/).
+
+```python
+import openai
+from langsmith import Client
+from langsmith.wrappers import wrap_openai
+from langchain_experimental.data_anonymizer import PresidioAnonymizer
```

Comment:
Hmm idk if I'd recommend langchain_experimental in our production docs. I'd use presidio directly